src/delegation/DelegationDecoder.ts

```diff
@@ -33,7 +33,7 @@ interface IChainRootDelegation extends Codec {
 }
 
 export function decodeRootDelegation(
-  encoded: Option<Tuple> | Tuple
+  encoded: Option<Tuple>
 ): RootDelegationRecord | null {
   assertCodecIsType(encoded, [
     'Option<(H256,AccountId,bool)>',
```

Comment:
if you are changing the static types, make sure to update the arguments to `assertCodecIsType` as well